src/Util/ValidatorUtil.php

```diff
@@ -51,4 +51,41 @@ public static function isArrayOf(array $items, string $type): bool
 
         return true;
     }
+
+    /**
+     * Checks if input is list of correct trytes
+     *
+     * @param array $hashedArrays
+     *
+     * @return bool
+     */
+    public static function isArrayOfHashes(array $hashedArrays)
+    {
+        foreach ($hashedArrays as $hash) {
```

Comment:
Looks like duplicate logic somehow, how about this?

```php
$length = strlen($hash);
if ($length === 90 || $length === 81) && !self::isTrytes($hash, $length) {
    return false;
}
```

Reply:
It makes sense. The enhanced logical condition you mention is better than original appraoch!

Reply:
@Techworker, sorry. This logical condition you mention seems to be different from original approach.
I think it should be changed to this:

```php
if (($length === 90 || $length !== 90) && !self::isTrytes($hash, $length)) {
    return false;
}
```

Reply:
Well, `$length === 90 || $length !== 90` is always true :-)

Ok, lets write down what is needed:

 - A hash needs to be either 81 (standard hash) or 90 trytes (hash + checksum) long
 - It needs to consist of valid trytes (only when 90 or 81)

What about this:

```php
public static function isArrayOfHashes(array $hashes)
{
    foreach($hashes as $hash) {
        $length = \strlen($hash);
        // needs to be 81 or 90 and a valid hash
        if(!(($length === 90 || $length === 81) && isTrytes($hash, $length))) {
            // exit early
            return false;
        }
    }

    return true;
}
```

Reply:
@Techworker, thank you for your reply. All right. It makes sense.
I found this problem because my test cases is not passed.
And this modified approach can be passed in Travis CI build :+1:.

You can also check about `testIsArrayOfHashesWithValidHashedArrays` and `testIsArrayOfHashesWithInvalidHashedArrays` :smile: .